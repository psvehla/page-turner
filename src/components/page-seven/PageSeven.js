import React from 'react';
import bike from '../../images/7.jpg';

import './PageSeven.css';

function PageSeven(props) {
  return (
    <div><img src={bike} alt="bike" onClick={() => props.onClick()} /></div>
  );
}

export default PageSeven;
