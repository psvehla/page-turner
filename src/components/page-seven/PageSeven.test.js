import React from 'react';
import renderer from 'react-test-renderer';
import PageSeven from './PageSeven';

it('renders correctly', () => {

  const tree = renderer
    .create(<PageSeven />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
