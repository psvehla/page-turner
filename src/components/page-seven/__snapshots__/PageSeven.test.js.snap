// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="bike"
    onClick={[Function]}
    src="7.jpg"
  />
</div>
`;
